### ISSUE TITLE
Clicking on a picture under "People" doesn't load the picture.

### ISSUE THREAD

**Issue by RikkiBC** [nextcloud/photos#1633]
**Describe the bug**
Clicking on a picture under "People" doesn't load the picture (404 error in the console log reliably occurs instead).

**To Reproduce**
1. Go to Photos --> People
2. Click on a person.
3. Select an image partway through the selection.
4. Picture never loads - loader spinner spins endlessly. Browser Console displays a 404 for the image not being found. However, if you then click on the right arrow to go to the next picture, it'll happily load a new picture (but starting from the start of the pictures, not from where you selected).

**Expected behavior**
I expect the picture I clicked to open, not for it to never load, and then me attempting to move to the next picture taking me back to the start of all the pictures.

**Desktop (please complete the following information):**
 - OS: Ubuntu 22.04
 - Browser Firefox
 - Version 109.0.1

**Browser log**
```
XHR PROPFIND https://my.domain.name/remote.php/dav/files/MyUsername/files/MyUsername/zzz - Google Export 2022-10/Google Photos export 2022-10-04/Family/20220902_184814.jpg
[HTTP/2 404 Not Found 521ms]

Could not open file /files/MyUsername/zzz - Google Export 2022-10/Google Photos export 2022-10-04/Family/20220902_184814.jpg Error: Invalid response: 404 Not Found
[Viewer.vue:495:58]
```

EDIT:

**Additional Details**

Nextcloud version: 25.0.3
Photos app version: 2.0.1

**Comment by RikkiBC** [comment]
Honestly, I don't know how this app works... but:

`https://my.domain.name/remote.php/dav/files/MyUsername/files/MyUsername/zzz - Google Export 2022-10/Google Photos export 2022-10-04/Family/20220902_184814.jpg`

Doesn't that seem wrong? "/files" is appearing in that path twice, for one...

**Comment by RikkiBC** [comment]
It's also worth noting this doesn't seem to happen anywhere else... just in the pictures under People.

**Comment by RikkiBC** [comment]
Also worth noting that this doesn't happen with Memories.

**Comment by it-guy92** [comment]
Hey, i exactly got the same problem. If i click in Pictures => People on a photo, it wont load

**Comment by FadeFx** [comment]
This is a duplicate of my issue
https://github.com/nextcloud/photos/issues/1614

**Comment by marcelklehr** [comment]
This should be fixed in v25.0.4
